easybuild/easyblocks/p/python.py

```diff
@@ -156,6 +150,13 @@ def sanity_check_step(self):
         """Custom sanity check for Python."""
 
         pyver = 'python' + '.'.join(self.version.split('.')[:2])
+        shlib_ext = get_shared_lib_ext()
+
+        if '--with-tcltk' in self.cfg['configopts']:
```

Comment:
Well, it's a better way to check if Tkinter should be there or not; it gives the flexibility to add `--with-tcltk` to `configopts` in the easyconfig itself without having `Tk` as a dependency, e.g. to use the system Tk.